assets/src/components/viewHeader.tsx

```diff
@@ -36,8 +35,7 @@ const ViewHeader: ViewHeaderType = ({
     <div className="m-view-header">
       {backlinkToView &&
       backlinkToView !== OpenView.None &&
-      deviceType === "mobile" &&
-      featureIsEnabled("nav_beta") ? (
+      deviceType === "mobile" ? (
```

Comment:
question (non-blocking): Does this mean there is a behavior change on the front-end for mobile devices as part of this PR? Or is `featureIsEnabled("nav_beta")` always returning true at this point?

Reply:
(I totally didn't put two and two together on this when going through and mechanistically deleting the code related to the feature flag, so thanks for pointing it out.)